### PR TITLE
[Merged by Bors] - feat(RingTheory/Kaehler): The Kaehler differential module of polynomial algebras.

### PR DESCRIPTION
**Pull request by erdOne**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by fpvandoorn** [comment]
(don't forget to mark `awaiting-review` if it's ready for review)

File: Mathlib/RingTheory/Kaehler.lean [hunk 1]
```diff
@@ -687,4 +689,117 @@ theorem KaehlerDifferential.mapBaseChange_tmul (x : B) (y : Ω[A⁄R]) :
...
+        LinearMap.map_smul, Derivation.liftKaehlerDifferential_comp_D]
+      congr 1
+      induction a using MvPolynomial.induction_on
+      · simp only [MvPolynomial.derivation_C, map_zero]
+      · simp only [map_add, *]
+      · simp [*]
+
+/-- `{ dx | x ∈ σ }` forms a basis of the relative differential module
+of a polynomial algebra `R[σ]`. -/
+def KaehlerDifferential.mvPolynomialBasis (σ) :
```
Comment (riccardobrasca) [inline Mathlib/RingTheory/Kaehler.lean:722]: Can you add the instance that the module is free?

**Review by riccardobrasca — COMMENTED** [review]
Thanks!

bors d+

File: Mathlib/RingTheory/Kaehler.lean (hunk 1 above)
Comment (erdOne) [reply Mathlib/RingTheory/Kaehler.lean:722]: The instance is already added.

Comment (riccardobrasca) [reply Mathlib/RingTheory/Kaehler.lean:722]: Ops

**Pull request closed without merging**